packages/core/src/storage.ts

```diff
@@ -17,16 +17,20 @@ import type { Class, Doc, Ref } from './classes'
 import type { Tx } from './tx'
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-export type QuerySelector<T> = {
-  $in?: T[]
```

Comment:
Just make it optional. Why we need different ones?
<img width="641" alt="Screenshot 2021-06-30 at 17 50 47" src="https://user-images.githubusercontent.com/477235/123948539-b567cf80-d9cb-11eb-9d58-ea2d94b06f67.png">

Reply:
For restrict multiple queryselector